algorithms/date_event_extraction/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "date_event_extraction-algorithm-relationshipg3-api",
+  "version": "1.0.0",
+  "description": "Get the date and event for an entity by running the Algorithm on a set of sentences.",
+  "main": "index.js",
+  "scripts": {
+    "prestart": "npm install",
+    "start": "node index.js"
+  },
+  "keywords": [],
+  "license": "ISC",
+  "private": true,
+  "dependencies": {
+    "body-parser": "~1.0.1",
+    "child_process": "^1.0.2",
+    "clone": "^2.1.1",
+    "connect": "^3.2.0",
+    "deasync": "^0.1.9",
+    "express": "^4.0.0",
+    "fs": "0.0.1-security",
```

Comment:
this shouldn't be in here (as discussed here https://github.com/MusicConnectionMachine/RelationshipsG3/pull/24#discussion_r108437433)